src/targets/maven.ts

```diff
@@ -135,14 +147,34 @@ export class MavenTarget extends BaseTarget {
     return this.reduceConfig(settings);
   }
 
+  private getAndroidSettings(): AndroidFields {
+    if (
+      !this.config.android.distDirRegex ||
+      !this.config.android.fileReplaceeRegex ||
+      !this.config.android.fileReplacerStr
+    ) {
+      throw new ConfigurationError(
+        'Required Android configuration not found in configuration file. ' +
+          'See the documentation for more details.'
+      );
+    }
+    return {
+      android: {
+        distDirRegex: this.config.android.distDirRegex,
+        fileReplaceeRegex: this.config.android.fileReplaceeRegex,
+        fileReplacerStr: this.config.android.fileReplacerStr,
+      },
+    };
+  }
+
   /**
    * Checks whether the required software to run this target is available
    * in the system. It assumes the config for this target to be available.
    * If there's required software missing, raises an error.
    */
   private checkRequiredSoftware(): void {
     this.logger.debug(
-      `Checking if Maven CLI is available on ${this.mavenConfig.mavenCliPath}...`
+      `Checking if Maven CLI is available: ${this.mavenConfig.mavenCliPath}`
```

Comment:
Having the value as a separate argument has a different meaning and makes debugging easier, that's why I'm advocating for that. This makes a difference especially for objects.
```suggestion
      'Checking if Maven CLI is available:', this.mavenConfig.mavenCliPath
```